### DISCUSSION TITLE
How to utilize prepare mode & reume run?

### DISCUSSION THREAD

**Discussion by lastb0isct** [martymac/fpart#58] [Q&A]
I have run a prepare mode pass utilizing -p but when i try and run the -r resume run mode it says `Invalid option value loaded from resumed run: OPT_TOOL_PATH`

Is there something else that is needed to be done to get this to work properly?

**Comment by martymac** [comment]
Hello,

Sorry, this was a bug and is fixed in master branch (see commit e6fb280f and PR #44).

Best regards,

Ganael.

**Comment by lastb0isct** [comment]
Got it! Thanks!!
